[Doc Improvement][Change notification for transcript using graph API][3099832]

v-sdhakshina opened the pull request.

Anjali-Patle commented on concepts/transcripts.md:
I think this is supposed to be in JSON format in other files. Can you please check?

Anjali-Patle commented on concepts/transcripts.md:
````diff
@@ -0,0 +1,101 @@
+
+The following is the payload for notifications without resource data and for a transcript being available for an online meeting.
+
+```http
````
I think this is supposed to be in JSON format in other files. Can you please check?

v-sdhakshina replied:
Updated as suggested

Anjali-Patle commented on concepts/transcripts.md:
````diff
@@ -0,0 +1,101 @@
+  } 
+} 
+```
+
````
Can you please add this line "The resource and @odata.id properties can be used to make calls to Microsoft Graph to get the transcript."  here, as per the document?

v-sdhakshina replied:
Updated as suggested

JarbasHorst commented on concepts/toc.yml:
```diff
@@ -238,6 +238,8 @@ items:
                     href: teams-changenotifications-chat.md
                   - name: Chat membership
                     href: teams-changenotifications-chatMembership.md
+                  - name: Transcripts
```
For consistency and clarity, please rename this TOC entry. Suggested revision:

- Call transcripts

After renaming, please also ensure that this TOC entry appears in the correct alphabetical order.

v-sdhakshina replied:
Updated as suggested

JarbasHorst commented on concepts/toc.yml:
```diff
@@ -238,6 +238,8 @@ items:
                   - name: Chat membership
                     href: teams-changenotifications-chatMembership.md
+                  - name: Transcripts
+                    href: transcripts.md
```
For clarity and consistency, please rename the file. Suggested revision:
- teams-changenotifications-calltranscript.md

After updating the filename, please update the reference links accordingly.

v-sdhakshina replied:
Updated as suggested

v-sdhakshina commented on concepts/transcripts.md:
```diff
@@ -0,0 +1,113 @@
+
+### Permissions
+
+One of the following permissions is required to subscribe to `communications/onlineMeetings/getAllTranscripts`. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
```
Updated as suggested

v-sdhakshina commented on concepts/transcripts.md:
```diff
@@ -0,0 +1,113 @@
+
+### Example
+
+The following example shows how to subscribe to transcripts available at tenant-level.
```
Updated as suggested

v-sdhakshina commented on concepts/transcripts.md:
```diff
@@ -0,0 +1,113 @@
+
+### Permissions
+
+One of the following permissions is required to subscribe to `communications/onlineMeetings/{onlineMeetingId}/transcripts`. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
```
Updated as suggested

v-sdhakshina commented on concepts/transcripts.md:
```diff
@@ -0,0 +1,113 @@
+
+### Example
+
+The following example shows how to subscribe to transcripts available for a particular online meeting.
```
Updated as sugegsted

JarbasHorst commented:
Hi @Lauragra, this PR is ready for final review. Please, merge PR [22027](https://github.com/microsoftgraph/microsoft-graph-docs/pull/22027) into this branch to include my final edits.

@v-sdhakshina, I reverted some of your edits in the teams-changenotifications-calltranscript.md file, so they align with our standards. Thanks!

JarbasHorst commented:
Hi @v-sdhakshina, please merge PR [22093](https://github.com/microsoftgraph/microsoft-graph-docs/pull/22093) to include the missing changelog record. Thanks!

JarbasHorst commented:
Hi @v-sdhakshina, I reviewed your PR and left one additional comment. Please take a look when you will have time.

**Also, to avoid merge conflicts, please merge PR [22093](https://github.com/microsoftgraph/microsoft-graph-docs/pull/22093) into your branch to include my fixes and before addressing my new comment.**

v-sdhakshina commented on concepts/includes/change-notifications-subscription-lifetime.md:
```diff
@@ -12,25 +12,26 @@ ms.localizationpriority: high
 
```
Added the doc as you suggested

v-sdhakshina commented:
Hi @JarbasHorst , I've added change-notifications-rich-notifications-supported-resources file and added Transcript information. Request you to please approve the PR.

JarbasHorst commented:
Hi @Lauragra this PR is ready for final review. Please, merge PR [22239](https://github.com/microsoftgraph/microsoft-graph-docs/pull/22239) into this branch to include my final edits.

Lauragra commented on concepts/teams-changenotifications-callrecording-and-calltranscript.md:
```diff
@@ -0,0 +1,113 @@
+
+This article describes scenarios for the **transcript** resource. For more details, see [Change notifications for Microsoft Teams resources](teams-change-notification-in-microsoft-teams-overview.md).
+
+## Subscribe to transcripts available at tenant-level
```
Change "at tenant-level" to "at the tenant level".

v-sdhakshina replied:
Updated as suggested

Lauragra commented on concepts/teams-changenotifications-callrecording-and-calltranscript.md:
```diff
@@ -0,0 +1,113 @@
+
+## Subscribe to transcripts available at tenant-level
+
+To get change notifications for any transcript available for any online meeting in a tenant, subscribe to `communications/onlineMeetings/getAllTranscripts`. The notification for a transcript is sent only if the subscription happens before the transcription starts. This subscription is supported only for regular scheduled meetings.
```
What type of meeting is not supported? Do you mean calls that aren't scheduled on the calendar? It would be good to make that clear here.

v-sdhakshina replied:
We only support one case for this notification i.e meeting scheduled on the calendar. Since there are multiple meeting types, we limited ourselves with supported meeting type.

Lauragra commented on concepts/teams-changenotifications-callrecording-and-calltranscript.md:
```diff
@@ -0,0 +1,113 @@
+
+To get change notifications for any transcript available for any online meeting in a tenant, subscribe to `communications/onlineMeetings/getAllTranscripts`. The notification for a transcript is sent only if the subscription happens before the transcription starts. This subscription is supported only for regular scheduled meetings.
+
+### Permissions
```
Please remove this section as we will be automating the permissions information in the reference docs. We don't want to have manual permissions tables that can get out of sync with the reference. You can replace with a link to the getAllTranscripts API topic for details.

Anjali-Patle replied:
@Lauragra the permissions and behavior of change notifications and Get All Transcripts API is different. In this case, the notification at tenant level is not supported in delegated context. However, the APIs support both delegated and app-context requests. So it would be difficult to keep a common permissions table for both.

Lauragra commented on concepts/teams-changenotifications-callrecording-and-calltranscript.md:
```diff
@@ -0,0 +1,113 @@
+
+To get change notifications for any transcript available for a particular online meeting, subscribe to `communications/onlineMeetings/{onlineMeetingId}/transcripts`. The notification for a transcript is sent only if the subscription happens before the transcription starts. This subscription is supported only for regular scheduled meetings.
+
+### Permissions
```
Same comment as previous regarding removing the permissions table and linking to the API topic.

v-sdhakshina replied:
Same as previous comment

Lauragra commented on api-reference/beta/api/subscription-delete.md:
```diff
@@ -31,44 +31,45 @@ Depending on the resource and the permission type (delegated or application) req
 
 | Supported resource | Delegated (work or school account) | Delegated (personal Microsoft account) | Application |
 |:-----|:-----|:-----|:-----|
```
Global: These topics have a deprecation notice (line 16 in this topic); should that be removed now?

v-sdhakshina replied:
Removed as suggested

v-sdhakshina commented on concepts/teams-changenotifications-callrecording-and-calltranscript.md:
```diff
@@ -0,0 +1,113 @@
+ms.custom: scenarios:getting-started
+---
+
+# Get change notification for meeting transcripts using Microsoft Graph
```
Updated as suggested

v-sdhakshina commented on concepts/teams-changenotifications-callrecording-and-calltranscript.md:
```diff
@@ -0,0 +1,113 @@
+
+# Get change notification for meeting transcripts using Microsoft Graph
+
+Change notifications enable you to subscribe to changes (create) to transcripts. You can get notified whenever a [transcript](/graph/api/resources/calltranscript) is available after an online meeting.
```
We've removed the word Create.

v-sdhakshina commented:
Hi @Lauragra , updated the doc as per your comments. Request you to please review and approve this PR, as we're planning to publish the docs today.

v-sdhakshina commented:
Hi @Lauragra , Request you to please merge the PR to live.
